BrainPortal/app/controllers/groups_controller.rb

```diff
@@ -78,7 +78,7 @@ def index  #:nodoc:
   # GET /groups/1.xml
   # GET /groups/1.json
   def show #:nodoc:
-    @group = current_user.public_and_available_groups.find(params[:id])
+    @group = current_user.viewable_groups.find(params[:id])
```

Comment:
I have an issue with this one. 

Does a SiteManager can show/edit a User project ? With a SiteManager I am able to see a User Project (belong to one user of the Site) but when I want to go to the show/edit page of this group I have the following issue: 
> ActiveRecord::RecordNotFound

Reply:
The `can_be_accessed_by` return false in this kind of situation.

Reply:
I will also change this back to assignable_groups to be compatible with the original behavior of available_groups